src/js/Framework/GoBack.js

```diff
@@ -0,0 +1,14 @@
+export class GoBack {
+  constructor () {
+    this.initGoBack()
+  }
+
+  initGoBack () {
```

Comment:
Ik zou de classes proberen gelijk te houden met wat er al in zit van js, dus event listeners allemaal in 1 method steken zoals https://github.com/sumocoders/FrameworkStylePackage/blob/master/src/js/Framework/LoadingBar.js#L6